docs/contributing/development.rst

```diff
@@ -82,30 +82,49 @@ Similarly, substantial patches with significant implications for the codebase
 from other members of the community should be reviewed and discussed by more
 than one core developer before being accepted.
 
+
 Current core developers
 ===============================
 
 * Angelo Dini http://github.com/finalangel
-* Benjamin Wohlwend http://github.com/piquadrat
 * Daniele Procida http://github.com/evildmp
 * Iacopo Spalletti http://github.com/yakky
 * Jonas Obrist http://github.com/ojii
 * Martin Koistinen http://github.com/mkoistinen
-* Paulo Alvarado http://github.com/czpython
 * Patrick Lauber http://github.com/digi604
+* Paulo Alvarado http://github.com/czpython
 * Stefan Foulis http://github.com/stefanfoulis
+* Vadim Sikora https://github.com/vxsx
+
 
```

Comment:
Suggestion: add a section here for "core designers" for Christian and Matthias (and/or any future core designers)